src/Components/Organisms/RegisterModal/RegisterModal.tsx

```diff
@@ -56,19 +60,26 @@ const RegisterModal = (prop: { open: boolean; handleClose: () => void }) => {
         <ClosingIcon handleClose={prop.handleClose} />
         <Formik
           initialValues={{
-            name: '',
             email: '',
             password: '',
             confirmPassword: '',
             termsOfService: false,
           }}
           validationSchema={RegisterValidation}
-          onSubmit={(values, { setSubmitting }) => {
-            setTimeout(() => {
-              setSubmitting(false);
-              const data = JSON.stringify(values, null, 2);
-              console.log(data);
-            }, 500);
+          onSubmit={async (values, { setSubmitting }) => {
+            setSubmitting(false);
+            try {
+              const credentialObj = await createUserWithEmailAndPassword(
+                auth,
+                values.email,
+                values.password,
+              );
+              console.log(credentialObj.user);
+              alert('Rejestracja udana');
```

Comment:
Maybe just a simple modal here could be better ?

Reply:
could be new task, not logic-issue only